packages/editor/src/reducers/index.ts

```diff
@@ -112,7 +146,23 @@ export const editorReducer = (
                 labelMapping: state.labelMapping,
                 modelMapping: state.modelMapping,
                 resources: state.resources,
-                identifyingProperty: state.identifyingProperty
+                identifyingProperty: state.identifyingProperty,
+                containersProperties: state.containersProperties
+            };
+        case ADD_CONTAINER_PROPERTIES:
+            if (_.isEmpty(action.containerProperties) ||
```

Comment:
Are there test cases that explicitly check whether the `containerProperites` have been updated?